plugins/packages/oracledb/package.json

```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "@tooljet-plugins/common": "file:../common",
     "knex": "^0.95.14",
-    "oracledb": "^5.3.0",
```

Comment:
This change may not be required.

Reply:
We can directly use the prebuilt binary so that for arm machines to run locally it can work out.
https://oracle.github.io/node-oracledb/INSTALL.html#-2-quick-start-node-oracledb-installation